pom.xml

```diff
@@ -266,23 +201,21 @@
       <version>1.16</version>
     </dependency>
 
-
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-all</artifactId>
+      <artifactId>mockito-core</artifactId>
       <version>1.10.19</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
-      <version>4.4.1</version>
+      <version>4.5</version>
```

Comment:
BTW this would be quite welcome since when having _other_ plugins depend on `git` in `test` scope, I otherwise need to `exclude` `httpclient` from the dependency.